CHANGELOG.md

```diff
@@ -26,6 +26,10 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
   Please report any remaining issues you have with the Git
   subprocessing path.
 
+* `ui.default-description` has been deprecated, and will be removed
+  in a future release. Please migrate to [`templates.draft_commit_description`](docs/config.md#default-description)
+  and/or [`templates.commit_trailers`](docs/config.md#commit-trailers) instead.
```

Comment:
Should we explicitly specify a release that this will be dropped in?

Reply:
I don't think we need the release number in changelog, but adding `TODO: Delete in jj x.xx` comment will help.

Reply:
When should we deprecate this? I read six releases somewhere, but unsure if that's universal or just on a case-by-case basis.

Reply:
The 6-month rule seems good for this. There are alternative options the user can switch to.